From 2fbb518bec6390339e3aff67093f30397fefb2de Mon Sep 17 00:00:00 2001
From: Aditya Garg <gargaditya08@live.com>
Date: Thu, 9 Oct 2025 10:53:52 +0530
Subject: [PATCH] Add T2 support

---
 Makefile     | 2 +-
 debian/rules | 6 ++++++
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 3ce7450..5aaad82 100644
--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,7 @@ KERNEL_MAJMIN=$(KERNEL_MAJ).$(KERNEL_MIN)
 KERNEL_VER=$(KERNEL_MAJMIN).$(KERNEL_PATCHLEVEL)
 
 DEB_VERSION=$(KERNEL_VER)-$(KREL)$(PKG_REV_EXTRA)
-EXTRAVERSION=-$(KREL)$(KREL_EXTRA)-pve
+EXTRAVERSION=-$(KREL)$(KREL_EXTRA)-pve-t2
 KVNAME=$(KERNEL_VER)$(EXTRAVERSION)
 PACKAGE=proxmox-kernel-$(KVNAME)
 HDRPACKAGE=proxmox-headers-$(KVNAME)
diff --git a/debian/rules b/debian/rules
index 956db63..933da65 100755
--- a/debian/rules
+++ b/debian/rules
@@ -106,6 +106,12 @@ PMX_CONFIG_OPTS= \
 -d CONFIG_N_GSM \
 -d UBSAN_BOUNDS \
 -d KVM_INTEL_PROVE_VE \
+-m CONFIG_HID_APPLETB_BL \
+-m CONFIG_HID_APPLETB_KBD \
+-m CONFIG_DRM_APPLETBDRM \
+-m CONFIG_BT_HCIBCM4377 \
+-m CONFIG_APPLE_BCE \
+-m CONFIG_APFS_FS \
 
 
 debian/control: $(wildcard debian/*.in)
-- 
2.32.7

